.env.template

```diff
@@ -17,3 +17,7 @@ CONTEXT_WINDOW_TOKENS=16000
 # MAX_RESPONSE_TOKENS
 # The maximum size of the LLM's response
 MAX_RESPONSE_TOKENS=2000
+
+# BRAVE_API_KEY
+# Get it from https://api.search.brave.com
+BRAVE_API_KEY=
```

Comment:
NIT: add a comment stating that this is needed for the `webScraper.search` script